packages/react-components/src/TextArea.tsx

```diff
@@ -0,0 +1,144 @@
+// Copyright 2017-2020 @polkadot/react-components authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+import React, { TextareaHTMLAttributes, useCallback } from 'react';
+import styled from 'styled-components';
+
+import { Labelled } from '.';
+import { ThemeProps } from './types';
+
+interface Props {
+  children?: React.ReactNode;
+  className?: string;
+  help?: React.ReactNode;
+  isError?: boolean;
+  isReadOnly?: boolean;
+  label?: React.ReactNode;
+  onChange?: (arg: string) => void;
+  seed?: string;
+  withLabel?: boolean;
+}
+
+interface TextInputProps extends TextareaHTMLAttributes<any>{
+  withError?: boolean;
+}
+
+const TextInput = ({ withError, ...textAreaProps }: TextInputProps) => <textarea
+  {...textAreaProps}
+  className={[textAreaProps.className, withError ? 'ui-textArea-withError' : ''].join(' ')}
+/>;
+
+const TextAreaInput = styled(TextInput)(({ theme }: ThemeProps) => `
+  background: ${theme.bgInput};
+  border-radius: 4px;
+  border: 1px solid #DDE1EB;
+  box-sizing: border-box;
+  color: ${theme.color};
+  display: block;
+  font-size: 1.15rem;
```

Comment:
```suggestion
```
Would suggest not having an override, it actually looks inconsistent with the rest of the UI.